mod-orders/mod-orders.postman_collection.json

```diff
@@ -7284,6 +7389,17 @@
 					"        }",
 					"        });  ",
 					"    };",
+					"    ",
+					"    /**",
+					"     * Deletes Holdings linked by PO lines from Inventory",
+					"     */",
+					"    utils.deleteHoldingRecords = function(order) {",
+					"      order.compositePoLines.forEach(function(line) {",
+					"        if (line.instance_id) {",
+					"          utils.processDeleteRequest(\"/holdings-storage/holdings?query=instanceId=\" + line.instance_id, \"Holding should be successfully deleted\");",
```

Comment:
According to [Holdings Storage API documentation](https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/holdings-storage.html#holdings_storage_holdings_delete) only `lang` parameter is supported so the `query` is ignored. Also as far as I understand when `DELETE /holdings-storage/holdings` endpoint is called, all the holdings are being deleted.